src/main/webapp/package.json.old

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I think it should be ok to delete this file, because you already have 1 package.json in the webapp folder

Reply:
Done